amazon-chime-sdk/src/main/java/com/amazonaws/services/chime/sdk/meetings/audiovideo/audio/AudioMode.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazonaws.services.chime.sdk.meetings.audiovideo.audio
+
+/**
+ * [AudioMode] describes the audio mode in which the audio client should operate during a meeting session.
```

Comment:
We need to be more specific. We need to mention that it affect the sample rate and the channel count fo the output (speaker) and the sample rate of the input (michrophone). Maybe we need to mention the sample rate and channel count values for input and output in every value aka Mono16K, Mono48K etc.

Reply:
Added more descriptions about audio mode.